recipes/libusb/all/conanfile.py

```diff
@@ -40,6 +40,11 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def build_requirements(self):
+        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ and \
```

Comment:
@madebr @SSE4 Should we consider it by default in all recipes?

Reply:
Do you mean removing the `CONAN_BASH_PATH` and `tools.os_info.get_windows_subsytem()` part?
I don't use conan in a msys2 terminal but I suppose some people do.

Reply:
@uilianries opening an issue to discuss this

Reply:
done https://github.com/conan-io/conan-center-index/issues/1059